examples/chip-tool/commands/common/Commands.cpp

```diff
@@ -45,9 +44,6 @@ int Commands::Run(int argc, char ** argv)
     err = mStorage.Init();
     VerifyOrExit(err == CHIP_NO_ERROR, ChipLogError(Controller, "Init Storage failure: %s", chip::ErrorStr(err)));
 
-    err = chip::GroupTesting::InitGroupData();
```

Comment:
If this is removed, Test Group commands will fail in Chip-tool. Why do we need to remove this ? I would leave it for now until group commands are fully implemented in Chip-tool

Reply:
@jepenven-silabs Doesn't the code in `CHIPCommand::Run` do the equivalent thing?

Are there particular steps that should be tested to make sure test group commands work?

Reply:
The group data initialization is now done in `CHIPCommand.cpp`. This was necessary because the key generation now must be done after the fabric table initialization.